From 8df19fa5a9ecd0162698fde8307f6a48df255b99 Mon Sep 17 00:00:00 2001
From: "yi.liu" <yi.liu@amlogic.com>
Date: Mon, 31 Jul 2023 10:01:17 +0800
Subject: [PATCH 5/5] amlbian: A311d deleltes isp service installed by amlbian
 [1/1]

PD#SWPL-133853

Problem:
Repeated installation of camera_isp_3a_server.service.

Solution:
Install isp service by isp.deb builded by yocto.

Verify:
local

Change-Id: I9f881f051b97c9cf99005ec9dbbeff01916cf8b0
Signed-off-by: yi.liu <yi.liu@amlogic.com>
---
 config/boards/amlogic/A311D-W400.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/boards/amlogic/A311D-W400.conf b/config/boards/amlogic/A311D-W400.conf
index b5f12c2..e6b081e 100644
--- a/config/boards/amlogic/A311D-W400.conf
+++ b/config/boards/amlogic/A311D-W400.conf
@@ -78,7 +78,7 @@ tweaks_platform() {
 
 	#Guoping TODO, need open config in kernel build, then enable service here, need change later
 	if [ "$LINUX" != "mainline" ]; then
-		eval 'LC_ALL=C LANG=C chroot $ROOTFS_TEMP /bin/bash -c "systemctl --no-reload enable bluetooth-amlbian.service  camera_isp_3a_server.service"'
+		eval 'LC_ALL=C LANG=C chroot $ROOTFS_TEMP /bin/bash -c "systemctl --no-reload enable bluetooth-amlbian.service"'
 	fi
 
 	if [ "$DISTRIB_TYPE" != "server" ] && [ "$LINUX" != "mainline" ]; then
-- 
2.25.1

